docs-2.0/3.ngql-guide/6.funtions-and-expressions/5.case-expressions.md

```diff
@@ -0,0 +1,161 @@
+# CASE expressions
+
+The `CASE` expression uses conditions to filter the result of an nGQL query statement. It is used in the `YIELD` or `RETURN` clause. nGQL provides two forms of `CASE` expressions just like openCypher: the simple form and the generic form.
+
+The `CASE` expression goes through conditions and returns a result when the first condition is met. Then the `CASE` expression stops reading the conditions and returns the result. If no conditions are met, it returns the result in the `ELSE` clause. If there is no `ELSE` clause and no conditions are met, it returns `NULL`.
+
+The following graph is used for the examples in this topic.
+
+![Example graph for CASE expressions](https://docs-cdn.nebula-graph.com.cn/docs-2.0/3.ngql-guide/6.funtions-and-expressions/case-example.png)
+
+## The simple form of CASE expressions
+
+### Syntax
+
+```nGQL
+CASE <comparer>
```

Comment:
can I use the alias as the <comoarer> here

Reply:
Sorry, I don't understand. What alias do you want to use? Do you mean using "alias" instead of "comparer"?

Reply:
I now get what you mean. We can't use the alias of a property in the CASE expression for now.